src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeRiskLevelCellConfigurator.swift

```diff
@@ -46,20 +48,29 @@ class HomeRiskLevelCellConfigurator: HomeRiskCellConfigurator {
 		}
 	}
 
+	private var buttonTitle: String {
```

Comment:
so I remember this part of code very vividly because I adjusted it and rewrote it in a previous pull request. i also know that there is multiple parts in the code where this method is used (afaik almost) copy-pasted every single time. So, it would be great if you can check this, and find out, whether you can extract it to a protocol with default methods or an extension, or really, any other method of reducing code duplication.